tests/unit/test_cli.py

```diff
@@ -15,7 +17,54 @@ def test_show_version(capsys):
   Executable location = (.+)
   Drools_jpy version = \d+\.\d+\.\d+
   Java home = (.+)
-  Java version = \d+\.\d+\.\d+
+  Java version = \d+\.\d+\.\d+(\.\d+)?
   Python version = \d+\.\d+\.\d+ (.+)$"""
     )
     assert pattern.match(output.out)
+
+
+def test_check_jvm_bad_java_home():
+    @patch("ansible_rulebook.util.get_java_home")
+    def get_java_home(mock_response):
+        mock_response.return_value = None
+
+        with pytest.raises(SystemExit) as excinfo:
+            check_jvm()
+            assert excinfo.value.code == 1
+
+
+@pytest.mark.parametrize(
+    "mocked_version",
+    [
+        pytest.param(
```

Comment:
shouldn't them be just one line?

Reply:
yeah, due to a extra comma, the formatter trolled me. Fixed